docker-compose-mock.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.7'
+
+services:
+
+  seq:
+    container_name: seq
+    image: datalust/seq:latest
+    environment:
+      - ACCEPT_EULA=Y
+    ports:
+      - 5341:80
+
+  jaeger:
+    container_name: jaeger
+    image: jaegertracing/opentelemetry-all-in-one:latest
+    ports:
+      - 13133:13133
+      - 16686:16686
+      - 4317:55680
+
+  brpproxy:
+    container_name: brpproxy
+    image: iswish/haal-centraal-brp-bevragen-proxy:2.0.15
```

Comment:
in de e2e docker-compose wordt de image haalcentraal.azurecr.io/brpproxy gebruikt. is er een reden dat er hier een andere bron voor de image is?

Reply:
waarom neem je hier een specifieke versie op i.p.v. :latest?